src/pages/home/report/LinkPreviewer.js

```diff
@@ -103,13 +105,13 @@ function LinkPreviewer(props) {
                             )}
                         </View>
                         {!_.isEmpty(title) && (
-                            <Text
+                            <TextLink
```

Comment:
We replaced the `Text` component to `TextLink` to make it clickable, but we didn't check the clickable area for the `pv2` padding and full width area. This caused a regression [here](https://github.com/Expensify/App/issues/22979).